src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -121,12 +114,7 @@ unsigned uct_rc_mlx5_iface_srq_post_recv(uct_rc_mlx5_iface_common_t *iface)
             srq->free_idx  = next_index;
         }
 
-        for (i = 0; i < iface->tm.mp.num_strides; ++i) {
-            if (seg->srq.ptr_mask & UCS_BIT(i)) {
-                /* Can reuse existing desc */
-                continue;
-            }
-
+        ucs_for_each_bit(i, (~seg->srq.ptr_mask & UCS_MASK(iface->tm.mp.num_strides))) {
```

Comment:
pls precalculate the bitmap before the loop

Reply:
done